samples/cpp/tutorial_code/objectDetection/objectDetection.cpp

```diff
@@ -44,7 +55,7 @@ int main( void )
         //-- 3. Apply the classifier to the frame
         detectAndDisplay( frame );
 
-        int c = waitKey(10);
+        int c = waitKey(1);
```

Comment:
i think `detectAndDisplay( frame );` need some time to precess, i did not measured it but i think it is over 30ms